src/ctirs/api/v1/gv/views.py

```diff
@@ -892,39 +742,26 @@ def get_stix_files_id(request, package_id):
 
 
 # DELETE /api/v1/gv/stix_file/<package_id>
-# stix_file 削除
 def delete_stix_files_id(request, package_id):
```

Comment:
No decorator.

Reply:
`delete_stix_files_id` calls from `stix_files_id` only.
`stix_files_id` has `@api_key_auth`.
So authentication is complete at this stage.